### ISSUE TITLE
Unable to auto populate the 'amount' field when calling transfer

### ISSUE THREAD

**Issue by devhalls** [skalenetwork/metaport#108]
**Description**
When passing 'interfaces.TransferParams' to the transfer method the amount field isn’t populating and has an empty value in the widget:
```
metaport.transfer({
  amount: '250000000000000000', 
  chains: ['mainnet', 'staging-legal-crazy-castor'], 
  tokenType: 'eth', 
  tokenKeyname: 'eth', 
  lockValue: false
})
```

**To Reproduce**
1. Call the transfer method passing the above parameters 

**Expected behavior**
The 'amount' field in the widget should have the value passed to the transfer method.

**Desktop:**
 - Browser Chrome
 - Version 109.0.5414.119

**Additional context**
Type and chains and other fields populate as expected, only the 'amount' field does not. No errors are visible in the console.

**Comment by cstrangedk** [comment]
Please double check that metaport version is latest

**Comment by rubenbacanu** [comment]
I have the same problem. At the first click it loads and and then sets the chains, but the value remains 0. At a second click it populate the amount field with the wanted value. I tried to call the transfer() function twice, but it doesn't work.

**Comment by dmytrotkk** [comment]
closing, this issue was fixed in 1.1.0
